### DISCUSSION TITLE
Timeseries capabilities included in ADF

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#57] [Announcements]
Hi,
The ADF has been updated with timeseries of global means as well: https://github.com/NorESMhub/ADF/pull/2

See an exaple here:
https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/n1850.ne30_tn14.hybrid_fatessp.202401031_1_20_vs_n1850.ne30_tn14.hybrid_fatessp.202401007_1_20/website/html_img/mean_diag_TimeSeries.html

Still need to add categories

**Comment by MichaelSchulzMETNO** [comment]
So nice ! there is so much info in time series. Big fan.

**Reply by adagj** [reply]
Thanks. I added categories as well: https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/n1850.ne30_tn14.hybrid_fatessp.202401031_1_20_vs_n1850.ne30_tn14.hybrid_fatessp.202401007_1_20/website/html_img/mean_diag_TimeSeries.html

**Comment by adagj** [comment]
I have included emissions and column burden as integrated values. Then I had to remove the off set and scale factor from the variables. We can see how it works. As it is now, it differs a bit from how they do it at NCAR.
